xbmc/input/ButtonTranslator.cpp

```diff
@@ -87,7 +87,7 @@ static const ActionMapping actions[] =
     { "showsubtitles"            , ACTION_SHOW_SUBTITLES },
     { "nextsubtitle"             , ACTION_NEXT_SUBTITLE },
     { "cyclesubtitle"            , ACTION_CYCLE_SUBTITLE },
-    { "codecinfo"                , ACTION_SHOW_CODEC },
+    { "playerdebug"              , ACTION_PLAYER_DEBUG },
```

Comment:
see my other comment, actually wanted to keep it, but accidentally changed it for one input source (keyboard).